backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/exportimport/ImportVmTemplateCommandBase.java

```diff
@@ -168,11 +172,11 @@ protected boolean validate() {
             initImportClonedTemplate();
         }
 
-        VmTemplate duplicateTemplate = vmTemplateDao.get(getParameters().getVmTemplate().getId());
+        VmStatic duplicateTemplateOrVm = vmStaticDao.get(getParameters().getVmTemplate().getId());
```

Comment:
afair, this call never returns a template (although templates are stored within vm_static as well) - and if so, we need to keep also the previous call